### PR TITLE
Error404

### PR DESCRIPTION
**Pull request by parv15**
Error404 Page

### CODE REVIEW

File: client/src/pages/routes.tsx [hunk 1]
```diff
@@ -91,6 +92,7 @@ const App = () => {
         <Route path="/registration-form" element={<Signup />} />
         <Route path="/reset-password" element={<ForgotPassword />} />
         <Route path="/join-waitlist" element={<WaitListPage />} />
+        <Route path="/*" element={<Error404/>} />
```
Comment (Lalit3716) [inline client/src/pages/routes.tsx:95]: Change it to just '*'

File: client/src/pages/Error404/Error404.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
...0-6 3 3 0 0 0 0 6zm16-16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-32 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0-16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM49 36a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-32 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm32 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM33 68a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16-48a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0 240a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16 32a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16-64a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16-32a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm80-176a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16-16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm32 48a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16-16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0-32a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm112 176a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-16 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM17 180a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0 16a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm0-32a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16 0a3 3 0 1 0 0-6 3 3 0 0 0 0 6zM17 84a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm32 64a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm16-16a3 3 0 1 0 0-6 3 3 0 0 0 0 6z'%3E%3C/path%3E%3C/svg%3E");
```
Comment (Lalit3716) [inline client/src/pages/Error404/Error404.tsx:9]: Make this a separate component in the components directory and import it from there.

File: client/src/pages/Error404/Error404.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import { useNavigate } from 'react-router-dom';
+import React from 'react';
+import Error from '../../../public/Illustration/Error.svg';
```
Comment (Lalit3716) [inline client/src/pages/Error404/Error404.tsx:3]: Move this `Error.svg` inside `src` directory and put it in the same folder in`components` directory as the new component you will make(see next review comment)

**Review by Lalit3716 — COMMENTED** [review]
Hi, @parv15 Thanks for the PR! I have reviewed your PR can you make these changes and update your PR.

**Comment by parv15** [comment]
@Lalit3716  Resolved the comments.Pls check

**Comment by Lalit3716** [comment]
LGTM Thanks Parv merging it 🚀

**Comment by parv15** [comment]
@Lalit3716 if possible Squash this 3 commits and merge it .

**Pull request merged**